packages/core/src/types/config/index.ts

```diff
@@ -49,11 +49,20 @@ export type BannerAndFooter = {
   dts?: string;
 };
 
+export type Redirect = {
```

Comment:
add `redirect` to here to prevent passing this config to rsbuild 
![image](https://github.com/user-attachments/assets/4e12f858-27ed-4358-987e-abd981880d4a)